torch/distributed/nn/api/remote_module.py

```diff
@@ -35,6 +46,15 @@ def _create_module(module_cls, args, kwargs, module_interface_cls=None):
         )
     if module_interface_cls is not None:
         module = torch.jit.script(module)
+    if device:
+        d = torch.device(device)
+        if d.type == "cuda" and d.index >= torch.cuda.device_count():
+            raise RuntimeError(
+                "CUDA device index must be less than the number of available CUDA devices {}".format(
+                    torch.cuda.device_count()
+                )
+            )
+        module.to(d)
```

Comment:
Is this a false positive? Looks like it's complaining that this is not covered by tests? I'm assuming its probably because its happening on a different process?

Reply:
Yes, this actually is covered by the tests.